src/components/Hint/Hint.jsx

```diff
@@ -0,0 +1,38 @@
+// @flow
+import React, { type Node } from 'react';
+import cx from 'classnames';
+import { getDataAttrs } from '../../dataUtils';
+import type { Data } from '../../types';
+
+import styles from './style.scss';
+
+type Props = {|
+  children?: Node,
+  className?: string,
+  data?: Data,
+  disabled?: boolean,
+  error?: boolean,
+  style?: { [key: string]: any },
```

Comment:
I think this prop should be removed. If we can override the styles of the component itself, then what designs are we enforcing?

Reply:
We give a sane default, which is the right design. But I trust consumers to use this for things like layout or composition.
For example, if you're embedding it in a more complex component, you might need to address it from your CSS.